### PR TITLE
Added swaggerui shortcode instructions to /Shortcodes/index.md

### PR DESCRIPTION
**Pull request by theletterf**
As requested in https://github.com/google/docsy/pull/112#issuecomment-513054976

### CODE REVIEW

**Comment by LisaFC** [comment]
Thanks for this! Just waiting for the checks to re-run now that I've merged in the actual shortcode/template.

**Comment by theletterf** [comment]
Not sure why the check fails. Is there something I should have done differently?

**Comment by theletterf** [comment]
@LisaFC Checks are still red. :-/

**Comment by LisaFC** [comment]
I'll have a look next time I'm properly online (am on holiday...)

**Comment by LisaFC** [comment]
Aha, I think I see what's going on  - the example isn't actually formatted as a code snippet, it's just the shortcode, so because this isn't a swagger layout page the build falls over. Should work fine if the example is marked up as a code example.  Let me know if you need more help.

**Comment by theletterf** [comment]
No worries. Enjoy! :)

File: userguide/content/en/docs/Adding content/Shortcodes/index.md [hunk 1]
```diff
@@ -223,4 +223,10 @@ resources:
 | 2 | Command. One of `Fit`, `Resize` or `Fill`. See [Image Processing Methods](https://gohugo.io/content-management/image-processing/#image-processing-methods).
 | 3 | Processing options, e.g. `400x450`. See [Image Processing Options](https://gohugo.io/content-management/image-processing/#image-processing-methods).
 
+### swaggerui
 
+The `swaggerui` shortcode can be placed anywhere inside a `swagger` layout; it renders [Swagger UI](https://swagger.io/tools/swagger-ui/) using any OpenAPI YAML or JSON file as source, which can be hosted anywhere (for example, the root `/static` folder of Hugo).
+
+{{< swaggerui src="/openapi/openapi.yaml" >}}
```
Comment (LisaFC) [inline userguide/content/en/docs/Adding content/Shortcodes/index.md]:
```suggestion
```go-html-template
{{</* swaggerui src="/openapi/openapi.yaml" */>}}
```
```

**Comment by LisaFC** [comment]
Fixed! Thanks again.

**Pull request merged**